assets/scss/_kanvas-corner-popup.scss

```
@@ -13,18 +13,36 @@
 }
 
 .kanvas-corner-popup .popup {
+  --kanvas-top-gap:1rem;
+  --kanvas-bottom-gap: 3rem;
+  --kanvas-navbar-height: 6rem;
   position: fixed;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)